includes/polling/functions.inc.php

```diff
@@ -164,7 +166,7 @@ function poll_device($device, $options)
 
     if ($config['os'][$device['os']]['group']) {
         $device['os_group'] = $config['os'][$device['os']]['group'];
-        echo '('.$device['os_group'].')';
+        echo ' ('.$device['os_group'].')' . PHP_EOL;
```

Comment:
I'm not sure the PHP_EOL is needed here. It would result in a double line return when group is set, but single when it is not.